VTEX - Checkout Configuration API.json

```diff
@@ -0,0 +1,350 @@
+                        "type": "integer",
+                        "description": "The minimum purchase amount required for a shopper's cart to qualify for a purchase.",
+                        "example": 123
+                    },
+                    "maximumItemsInCart": {
+                        "type": "integer",
+                        "description": "The maximum number of items allowed in the cart per transaction.",
+                        "example": 10
+                    },
+                    "promoCode": {
+                        "type": "boolean",
+                        "description": "Indicates whether the cart allows promotional codes or discount coupons. Enabling this feature provides shoppers with the option to input promo codes during checkout.",
+                        "example": true
+                    },
+                    "shippingPreview": {
+                        "type": "boolean",
+                        "description": "Indicates whether the cart provides a preview of shipping costs before finalizing the purchase. Enabling this feature allows shoppers to view estimated shipping charges early in the checkout process.",
+                        "example": false
+                    },
+                    "skuPrice": {
+                        "type": "string",
+                        "description": "Specifies the pricing model used for displaying prices in the cart, either based on individual SKUs or the total price only.",
+                        "enum": [
+                            "TotalOnly",
+                            "SkuAndTotalPrice"
+                        ]
+                    }
+                }
+            },
+            "OrderReview": {
+                "type": "object",
+                "description": "Object of tooltips that provide additional information regarding the order review process. ",
+                "properties": {
+                    "taxesToolTip": {
+                        "type": "string",
+                        "description": "This tooltip may include details such as how taxes are calculated, applicable tax rates, or explanations about specific tax-related charges.",
+                        "example": "ToolTip"
+                    }
+                }
+            },
+            "Advanced": {
+                "type": "object",
+                "description": "Object containing advanced configuration of the checkout.",
+                "properties": {
+                    "adobeAnalyticsPlataformId1": {
+                        "type": "string",
+                        "description": "The Adobe Analytics Platform ID used for tracking analytics data.",
+                        "example": "adobeAnalyticsPlataformId1"
+                    },
+                    "adobeAnalyticsPlataformId2": {
+                        "type": "string",
+                        "description": "An additional Adobe Analytics Platform ID for tracking analytics data.",
```

Comment:
Add link to the Adobe Analytics platform and explain why you would need an additional ID